alacritty_terminal/src/term/mod.rs

```diff
@@ -1087,6 +1092,9 @@ impl<T> Term<T> {
 
         let colors = color::List::from(&config.colors);
 
+        let mut damage = Damage::new(num_lines, num_cols);
+        damage.expand_line(Line(0), Column(0), Column(0));
```

Comment:
What is the idea behind this call for `Line(0)`?

Reply:
Oh, ok. Would you mind adding a comment about it?

Reply:
So that a fresh term has its initial cursor position damaged.

It is unlikely to ever manifest itself in normal usage, but the tests revealed it.